### PR TITLE
Swizzle support for colors

### PR DESCRIPTION
**Pull request by DigiDuncan**
We talked about this in the Discord once. And why not, right? It's a cheap function to write.

Could probably have docs or something!

Takes a string of r, g, b, and a's and returns a tuple of those properties in order.

It's an [OpenGL thing.](https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)#Swizzling)

### CODE REVIEW

File: arcade/types.py [hunk 1]
```diff
@@ -419,6 +417,14 @@ def random(
 
         return cls(r, g, b, a)
 
+    def swizzle(self, swizzle_string: str) -> Tuple[int, ...]:
+        ret = []
+        for c in swizzle_string.lower():
+            if c not in 'rgba':
+                raise Exception("Swizzle string must only contain characters in [RGBArgba].")
```
Comment (pushfoo) [inline arcade/types.py]:
```suggestion
                raise ValueError("Swizzle string must only contain characters in [RGBArgba].")
```
I'd also consider outputting the offending value.

**Review by pushfoo — CHANGES_REQUESTED** [review]
This is almost ready. I think the current implementation is faster and easier than `__getattribute__` hacks, even if `color_instance.brga` would be cleaner code. 

Changes I'd like:
1. Use `ValueError` insteaad of `Exception`
2. Consider mentioning the offending value in the exception
3. Add tests

File: arcade/types.py (hunk 1 above)
Comment (DigiDuncan) [reply arcade/types.py]: Done the error improvement, tests to come!

Comment (DigiDuncan) [reply arcade/types.py]: Thanks, @pushfoo, for the tests!

**Review by pushfoo — APPROVED** [review]
This looks good now.

**Pull request merged**